### DISCUSSION TITLE
Frigate not logging events

### DISCUSSION THREAD

**Discussion by PANiCnz** [blakeblackshear/frigate#1337] [Ask A Question] [answered]
First time user of Frigate, so far its up and running. But in web gui when I got to the Events tab its not logging anything. Is it meant to? I had hoped it would provide a log of all the objects that had been detected.

Frigate appears to be successfully detecting objects as when I click on the camera it has recent examples displayed underneath.

Any advice?

**Comment by blakeblackshear** [comment, marked as answer]
You must have snapshots or clips enabled to get events.

**Reply by PANiCnz** [reply]
Thanks all sorted now. Had clips configured at a global level rather than at a camera level. All working now.
